src/tox/helper/build_isolated.py

```diff
@@ -4,7 +4,7 @@
 backend_spec = sys.argv[2]
 backend_obj = sys.argv[3] if len(sys.argv) >= 4 else None
 
-backend = __import__(backend_spec, fromlist=[None])
+backend = __import__(backend_spec)
```

Comment:
this should be `fromlist=['_trash']`

Reply:
👍 - Though also I think the "right" answer on Python 3.3 and above is to use `importlib.import_module`. So I'll split this into two branches, so that this `__import__` branch can be removed once support for old Python versions is removed.

Reply:
I disagree, please just use `__import__`, no need to involve a separate module to do exactly the same thing